README.md

```diff
@@ -145,9 +146,9 @@ If you *really* want to skip the linting during a commit, you can always run `gi
 
 ### Notes
 
-#### Jenkins
+#### Github Actions
 
-Jenkins automatically handles the generation of docker images.  Currently it is configured to trigger an image build for each push on individual branches.  The image generated will have the name `climate-explorer-frontend/[branch_name]`.
+The workflows setup in actions will automatically build, tag and publish to our [docker hub](https://hub.docker.com/r/pcic/climate-explorer-frontend).
 
 ##### Configuration, environment variables, and Docker
```

Comment:
I think this should be a four-hash headline too; it's not "under"  `Github Actions`, right?